### DISCUSSION TITLE
Overlay option

### DISCUSSION THREAD

**Discussion by davidmluque** [katspaugh/wavesurfer.js#3563] [Q&A] [answered]
Hello,

Less than a year ago I created a player for my company using wavesurfer 6.6.3. It looks like this:

![image](https://github.com/katspaugh/wavesurfer.js/assets/83181396/58fc108a-e3e5-49ed-bce5-a895139a0fab)

We are in the process of upgrading the application using new components and more updated versions of the libraries. The problem is that I have jumped to version 7 of wavesurfer, and the overlay option has disappeared, which converted two waves into one, but respecting the colors, which allowed me to separate the speakers so that it was clear when the agent was speaking and when the client was speaking.

This is how i created the wavesurfer (using VUE)

```js
this.wavesurfer = WaveSurfer.create({
    container: "#waveform",
    backend: 'MediaElement',                
    barWidth: 4,
    barHeight: 0.8,
    barMinHeight: 0,    
    splitChannels: true,                     
    splitChannelsOptions: {
        overlay: true,
        channelColors: {
            0: {progressColor:'#7D7D7D', waveColor:'#000'},
            1: {progressColor:'#7D7D7D', waveColor:'#000'},
        },
    },
    plugins: [ markersPlugin, cursor, timeline,regions ]
})
```
Is there any way to replicate this with wavesurfer version 7?

**Comment by katspaugh** [comment]
Unfortunately there's no such option in v7 indeed but it should be fairly easy to restore.
I've created an issue to restore it.

**Reply by davidmluque** [reply]
Thank you!!
I hope it won't be too complicated.

**Reply by katspaugh** [reply, marked as answer]
Sorry for the delay, the overlay option is now back, added in #3619 and will be released in 7.7.7.

**Reply by davidmluque** [reply]
This week I will implement the new version and test the split channels.

Thank you very much.
